.github/workflows/build_images.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build OCI Images
+
+on:
+  push:
+    branches:
+      - master
+      - gh_workflow
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the ci image
+        run: docker build . --file Containerfile --target ci-base --tag pulp/pulp-ci:latest
+      - name: Build the single-container image
+        run: docker build . --file Containerfile --tag pulp/pulp-fedora31:latest
+      - name: Greet on gh_workflow
+        if: ${{ branch == "gh_workflow" }}
+        run: echo "On branch gh_workflow; No pushing today!"
+      - name: Push ci image to dockerhub
+        if: ${{ branch == "master" }}
```

Comment:
```suggestion
        if: "contains(github.ref, 'master')"
```

Reply:
https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context